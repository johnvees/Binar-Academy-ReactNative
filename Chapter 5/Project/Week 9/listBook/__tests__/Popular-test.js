import * as React from 'react';
import {create} from 'react-test-renderer';

import Popular from '../src/components/Popular';
import ContainerTesting from '../src/utils/helpers/containerTesting';

describe('Popular Card Snapshot', () => {
  test('should render', () => {
    const component = create(ContainerTesting(<Popular />)).toJSON();
    expect(component).toMatchSnapshot();
  });
});
