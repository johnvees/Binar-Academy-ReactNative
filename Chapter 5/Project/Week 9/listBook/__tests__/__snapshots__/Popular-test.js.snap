// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popular Card Snapshot should render 1`] = `null`;
